ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -165,6 +165,11 @@ public void updateExtraData(ReactEditText view, Object extraData) {
     }
   }
 
+  @ReactProp(name = ViewProps.COLOR, defaultInt = 0, customType="color")
```

Comment:
We need to figure that out before we can take it.  It's possible for people to want to change their color and then back.  You could also look at why the color that works via spans doesn't for password colors.